Clarify that saving a Discover session always saves to the library (#6317)
elastic/docs-content: explore-analyze/discover/save-open-search.md

3. In **Add to dashboard**, select **New** to create a dashboard, or **Existing** to choose one from the list.
4. Select **Save and go to dashboard**.

#### Save a Discover session and add it to a dashboard [save-discover-session-with-add-to-dashboard]

From Discover, save the entire Discover session, including all its tabs, columns, sort order, and other settings.

::::{applies-switch}

:::{applies-item} {"stack": "ga 9.5", "serverless": "ga"}

> The session is saved to the library. In **Add to dashboard**, choose whether to also add it as a panel on a dashboard:

Author: ```suggestion
When you save a Discover session, it is saved to the library. The **Add to dashboard** option of the dialog lets you choose whether to also add it as a panel on a dashboard:
```

- **None**: The session is saved to the library only. To add it as a panel later, [add it from the dashboard library](#add-discover-session-from-library).
- **New**: The session is also added as a panel on a new dashboard.

> - **Existing**: The session is also added as a panel on a dashboard you choose.

Author: ```suggestion
- **Existing**: The session is also added as a panel on a dashboard you choose.
- **New**: The session is also added as a panel on a new dashboard.
- **None**: The session is saved to the library only. To add it as a panel later, [add it from the dashboard library](#add-discover-session-from-library).
```
ordering

2. Make your changes in **Discover**.
3. In the application menu, from the menu next to **Save and return**, select **Save as**.
4. In the **Save Discover session** dialog, enter a **Title** for the new session, and optionally a **Description** and [**Tags**](../find-and-organize/tags.md).
5. {applies_to}`stack: ga 9.5` {applies_to}`serverless: ga` In **Add to dashboard**, choose whether to also add the new session as a panel on a dashboard. The session is saved to the library in all cases.

   - **None**: The session is saved to the library only.
   - **New**: The session is also added as a panel on a new dashboard.

>    - **Existing**: The session is also added as a panel on a dashboard you choose.

Author: ```suggestion
   - **Existing**: The session is also added as a panel on a dashboard you choose.
   - **New**: The session is also added as a panel on a new dashboard.
   - **None**: The session is saved to the library only.
```
ordering
